Big_data_project.ipynb

```diff
@@ -0,0 +1,289 @@
+{
+  "nbformat": 4,
+  "nbformat_minor": 0,
+  "metadata": {
+    "colab": {
+      "provenance": [],
+      "collapsed_sections": [],
+      "mount_file_id": "1_4fIlT5IybjCCfHSLYHIEmnNvCUlt9gQ",
+      "authorship_tag": "ABX9TyMyqNepX1QTjJzSeV/cYxeE",
+      "include_colab_link": true
+    },
+    "kernelspec": {
+      "name": "python3",
+      "display_name": "Python 3"
+    },
+    "language_info": {
+      "name": "python"
+    }
+  },
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "view-in-github",
+        "colab_type": "text"
+      },
+      "source": [
+        "<a href=\"https://colab.research.google.com/github/PiotrMaciejKowalski/BigData2022-films/blob/load-data/Big_data_project.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>"
```

Comment:
Notatnik zwłaszcza colabowe powinny mieć dobrze określone sekcje. Sugeruje zrobić tak aby twój kod był w całości w sekcji pierwszej Spark Setup and Data Load, oraz dzielił się na subsekcje.